CHANGELOG.md

```diff
@@ -13,6 +13,8 @@
 
 ## Master
 
+- Fix excluded Swiftlint paths [@absolute-heike][] - [#180](https://github.com/danger/swift/pull/180)
```

Comment:
Note: When you add `[@absolute-heike] http://github.com/absolute-heike` at the end of the file then it will be linked/rendered properly.